src/sqlancer/stonedb/StoneDBOptions.java

```diff
@@ -0,0 +1,46 @@
+package sqlancer.stonedb;
+
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.Parameters;
+import sqlancer.DBMSSpecificOptions;
+import sqlancer.OracleFactory;
+import sqlancer.common.oracle.TestOracle;
+import sqlancer.mysql.MySQLOptions;
+import sqlancer.stonedb.StoneDBProvider.StoneDBGlobalState;
+import sqlancer.stonedb.oracle.StoneDBNoRECOracle;
+import sqlancer.stonedb.oracle.StoneDBTLPOracle;
+import sqlancer.stonedb.StoneDBOptions.StoneDBOracleFactory;
+
+import java.sql.SQLException;
+import java.util.Arrays;
+import java.util.List;
+
+@Parameters(separators = "=", commandDescription = "StoneDB (default host: " + MySQLOptions.DEFAULT_HOST
```

Comment:
Here, I think you should create a StoneDBOption, as it defines DBMS-specific parameters.

Reply:
Oh, I just noticed that you have a StoneDBOption class, but do not import anything from MySQLOption class. Define it in StoneDBOption class.